slides/section_1_setting_up_environment.md

```diff
@@ -341,9 +341,9 @@ Read through and follow along until the end of the episode page.
 rm -r venv/
 python3 -m venv venv
 source venv/bin/activate
-pip install <your_dependencies> 
+pip install --editable . <list_your_dependencies>
```

Comment:
```suggestion
pip install <list_your_dependencies>
```
I think this should stay what it was right?

Reply:
Yes, good catch! We don't introduce the `--editable` install until the next episode.